SEMINAR 8/model.py

```diff
@@ -0,0 +1,56 @@
+import csv
+
+def download_phonebook():
+    global id 
+    global name 
+    global department
+    global phone
+    id = []
+    name = []
+    department = []
+    phone = []  
+
+    with open('phonebook.csv', encoding='utf-8') as r_file:
+        file_reader = csv.reader(r_file, delimiter = ";")
+        count = 0
+        for row in file_reader:
+            if count == 0:
+                print(f'Файл содержит столбцы: {", ".join(row)}')
+            else:
+                print(f'{row[0]} сотрудник {row[1]} работает в\
+                {row[2]} отделе, его номер телефона {row[3]}')
```

Comment:
При выводе, получается, вот такой разрыв.
![image](https://user-images.githubusercontent.com/49741194/203688406-6959f6aa-8c84-4e9a-8aa6-d20e4bc803e8.png)
Можно сделать так, тогда будет все хорошо.
```suggestion
                print(f'{row[0]} сотрудник {row[1]} работает в'
                      f'{row[2]} отделе, его номер телефона {row[3]}')
```
![image](https://user-images.githubusercontent.com/49741194/203688540-54f99f5e-4be9-4767-8b28-ac1f661e5895.png)